app/services/idv/validate_document_capture_session.rb

```diff
@@ -0,0 +1,51 @@
+module Idv
+  class ValidateDocumentCaptureSession
+    include ActiveModel::Model
+
+    validates :session_uuid, presence: { message: 'session missing' }
+    validate :session_exists, if: :session_uuid_present?
+    validate :session_not_expired, if: :session_uuid_present?
+
+    def initialize(session_uuid)
+      @session_uuid = session_uuid
+    end
+
+    def call
+      @success = valid?
+
+      FormResponse.new(success: success, errors: errors, extra: extra_analytics_attributes)
+    end
```

Comment:
For my understanding: If it quacks like a form in that we validate a submission and return a `FormResponse`, should this live in `app/forms` ?

Reply:
oh good point! updated in 254f8fc46f254ab950c9fa47c27630cb682de990